docs/reference/licensing/start-trial.asciidoc

```diff
@@ -16,19 +16,17 @@ This API starts a 30-day trial license.
 [float]
 ==== Description
 
-The `start trial` API enables you to upgrade from a basic license to a 30-day
-trial license, which gives access to the platinum features.
+The `start trial` API enables you to upgrade from Basic to a 30-day trial, which
```

Comment:
Do we mention `Basic` elsewhere before this point? Since it is the default, users may not even know that they are running a basic license.

What about `to obtain a 30-day trial`, or something along those lines?

Reply:
Good point.  I thought maybe it was implying you couldn't use this API if you had a Gold+ subscription installed, but I agree it's better to simplify this paragraph.